docs/UserGuide.md

```diff
@@ -3,271 +3,478 @@ layout: page
-* Existing values will be updated to the input values.
+### List all clients : `list`
+
+**Purpose**: Shows a list of all clients in ClientBook. An optional attribute option can be added to show a list with only the specified attribute.
+
+**Format**: `list [-ATTRIBUTE]`
 
+**Examples**: 
+* Shows a list of all clients and all their information.
+  * `list`
+    
+    ![list](images/list.png)
+    
+*  Shows a list of all clients and their insurance policy number.
+   * `list -i`
 
-Examples:
-*  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
-*  `edit 2 n/Betsy Crower` Edits the name of the 2nd person to be `Betsy Crower`.
+    ![list policy](images/list-policy.png)
 
-### Locating persons by name: `find`
+*  Shows a list of all clients and their phone number and insurance policy number.
+   * `list -p -i`
+    
+     ![list phone](images/list-phone-policy.png)
 
-Finds and displays all clients whose field (name, phone, email, address, tags, insurance policy) contains any of the given keywords.
+[Return to Table of Contents](#table-of-contents)
 
-![find_alex_david](images/find_alex_david.png)
+  
+### Search for client contact based on keywords: `find`
 
-![find_alex_&_david](images/find_alex_&_david.png)
+**Purpose**: Finds and displays all clients whose field (name, phone, email, address, tags, insurance policy) contains any of the given keywords.
+An optional attribute option can be added to show the list of matched clients with only the specified attribute.
 
 ![find_alex_&_david_with_filter](images/find_alex_&_david_with_filter.png)
 
-Format: `find FLAG/KEYWORD [& MORE_KEYWORDS] [-ATTRIBUTES]…​`
+**Format**: `find FLAG/KEYWORD [& MORE_KEYWORDS] [-ATTRIBUTES]…​`
 
-* The search is case-insensitive. e.g hans will match Hans.
-* The order of the keywords does not matter.
+* The search is **case-insensitive**. 
+  * E.g. hans will match Hans.
+* The **order of the keywords does not matter**.
 * Only one `FLAG` can be used in each find command.
 * The `FLAG` can only be from: `n`, `p`, `e`, `a`, `t` and `i`, representing name, phone, email, address, tags and insurance policies respectively.
-* The delimiter `&` between keywords is used to search using multiple keywords. e.g. `Aaron & Tan` will return all persons with either `Aaron` or `Tan` in their names.
-* Without the use of delimiter `&`, all keywords following the `FLAG` will be used in the search. e.g. `Aaron Tan` will only return persons with `Aaron Tan` in their names.
-* Clients whose chosen field contains at least one keyword will be returned (i.e. OR search). e.g. Hans & Bo will return Hans Sum, Bo Yang.
-* Filter options are set by typing the attribute name after a dash. e.g. `-phone` filters phone number information of the listed clients
+* The delimiter `&` between keywords is used to search using multiple keywords. 
```

Comment:
A suggestion would be "The delimiter `&` between keywords allows you to search for Clients using multiple keywords."